README.md

```diff
@@ -9,18 +9,31 @@ This package contains a DNS provider module for [Caddy](https://github.com/caddy
 dns.providers.duckdns
 ```
 
+## Caddyfile definition
+
+```
+duckdns [<api_token>] {
+    api_token <api_token>
+    override_domain <duckdns_domain>
+}
+```
+
+`api_token` may be specified as an argument to the `duckdns` directive, or in the body.
+`override_domain` is optional; see "Challenge delegation" below.
```

Comment:
Oops, sorry, one thing:

```suggestion
- `api_token` may be specified as an argument to the `duckdns` directive, or in the body.
- `override_domain` is optional; see "Challenge delegation" below.
```

Markdown collapses them onto one line without this

Reply:
Changed (also neat, TIL GitHub has an inline Commit suggestion button)